src/pages/AnalyticsPage/AnalyticsPage.jsx

```diff
@@ -56,23 +57,25 @@ function AnalyticsPage() {
   }
 
   return (
-    <div className="mt-2 flex flex-col gap-4">
+    <div className={cn({ 'mt-2': !newHeader })}>
       {newHeader ? null : <Header />}
-      <div>{ownerData?.isCurrentUserPartOfOrg && <Tabs />}</div>
-      <ChartSelectors
-        params={params}
-        updateParams={updateParams}
-        active={true}
-        sortItem={sortItem}
-      />
-      <Suspense fallback={<SuspenseFallback />}>
-        <Chart params={params} />
-      </Suspense>
-      <ReposTable
-        owner={owner}
-        searchValue={params?.search}
-        filterValues={params?.repositories}
-      />
+      <div className="flex flex-col gap-4">
+        <div>{ownerData?.isCurrentUserPartOfOrg && <Tabs />}</div>
```

Comment:
should the div be wrapped in this conditional too? Otherwise we have a hanging div

I also think we prefer the Element ? Element : null syntax